packages/theme/src/ee/pages/dashboard/settings/roles/Index.vue

```diff
@@ -103,8 +103,8 @@ import DashboardPageHeader from "../../../../../components/dashboard/PageHeader.
 import BreadcrumbItem from "../../../../../components/ui/breadcrumbs/BreadcrumbItem.vue";
 
 const { settings } = useSettingStore()
-const { permissions  } = useUserStore()
-const dashboardRoles = useDashboardRoles();
+const { permissions } = useUserStore()
+const dashboardRoles = ref(useDashboardRoles());
```

Comment:
We don't need to warp the `dashboardRoles()` function call inside a `ref()`.

In case we're trying to access a value that needs reactivity that's a different story.